tests/unit/s2n_connection_test.c

```diff
@@ -126,7 +126,7 @@ int main(int argc, char **argv)
      */
     {
         /* Carefully consider any increases to this number. */
-        const uint16_t max_connection_size = 4190;
+        const uint16_t max_connection_size = 4222;
```

Comment:
By the way, the reason why FreeBSD is failing is because it measures the connection size as being a little larger. Here is what it thinks the connection size should be:
s2n_connection size (4224) no longer in (3799, 4222)

Reply:
Ah okay, thanks! I increased it in 64047cd7d483ac5e1be6e2d5845ab6752473e423.